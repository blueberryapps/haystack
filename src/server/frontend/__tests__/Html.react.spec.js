import { shallow } from 'enzyme';
import React from 'react';
import Html from '../Html.react';

const props = {
  bodyHtml: '<div />',
  javascripts: { app: 'app.xxxx.js' }
};

test('shallowly renders Html', () => {
  expect(shallow(<Html {...props} />)).toMatchSnapshot();
});
