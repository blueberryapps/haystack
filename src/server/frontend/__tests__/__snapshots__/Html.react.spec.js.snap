exports[`test shallowly renders Html 1`] = `
<html
  lang="en">
  <head>
    <meta
      charSet="utf-8" />
    <meta
      content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no"
      name="viewport" />
    <meta
      content="ie=edge"
      httpEquiv="x-ua-compatible" />
    <Rollbar />
  </head>
  <body>
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "<div />",
        }
      }
      id="app" />
    <Script
      src="app.xxxx.js" />
  </body>
</html>
`;
